ui/src/App.css

```
@@ -36,6 +36,79 @@
+  justify-content: center;
+  padding: 0;
+  font-size: 1rem;
+  line-height: 1;
+}
+
+.refresh-error-banner__retry:hover,
+.refresh-error-banner__dismiss:hover {
+  background: color-mix(in oklch, currentColor 12%, transparent);
+}
+
+@keyframes refreshErrorBannerIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "refreshErrorBannerIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)